includes/wikia/VariablesBase.php

```diff
@@ -6433,6 +6433,16 @@
  */
 $wgProfileToDatabase = false;
 
+/**
+ * The Prometheus Pushgateway exists to allow ephemeral and batch jobs to expose their metrics to Prometheus.
+ * Since these kinds of jobs may not exist long enough to be scraped, they can instead push their metrics
+ * to a Pushgateway. The Pushgateway then exposes these metrics to Prometheus.
+ *
+ * @see lib/Wikia/src/Metrics/Collector.php
+ * @var string $wgPrometheusPushgatewayHost
+ */
+$wgPrometheusPushgatewayHost = 'pushgateway.k8s.wikia.net';
```

Comment:
I've deployed proper pushgateway setup (one per namespace) so please update url to:
SJC: `pushgateway-prod.sjc.k8s.wikia.net`
RES: `pushgateway-prod.res.k8s.wikia.net`